src/renderers/WebGLRenderer.js

```diff
@@ -2421,6 +2421,8 @@ function WebGLRenderer( parameters ) {
 
 	this.setFramebuffer = function ( value ) {
 
+		if (_framebuffer !== value ) _gl.bindFramebuffer( _gl.FRAMEBUFFER, value );
+
```

Comment:
According to the behavior in `.setRenderTarget()`

https://github.com/mrdoob/three.js/blob/r103/src/renderers/WebGLRenderer.js#L2444-L2483

`_framebuffer` is used if `renderTarget` is `null`. So binding framebuffer here only if  `_currentRenderTarget` is `null`?

```javascript
if ( _framebuffer !== value && _currentRenderTarget === null ) _gl.bindFramebuffer( _gl.FRAMEBUFFER, value );
```

Reply:
Thanks. Request incorporated in the PR